sim_interface/source/sim_interface.cpp

```diff
@@ -1068,6 +1069,7 @@ Sim_output Sim_interface::simulate(const Sim_input& input)
         debug_topic += "DPS hamstring: " + std::to_string(dmg_dist.hamstring_damage * f) + "<br>";
         debug_topic += "DPS deep wounds: " + std::to_string(dmg_dist.deep_wounds_damage * f) + "<br>";
         debug_topic += "DPS item effects: " + std::to_string(dmg_dist.item_hit_effects_damage * f) + "<br><br>";
+        debug_topic += "DPS devastate: " + std::to_string(dmg_dist.devastate_damage * f) + "<br><br>";
```

Comment:
nit: I'd move devastate up a little, if it's used, it should be a sizable chunk of all damage.